src/index.ts

```diff
@@ -229,21 +270,23 @@ class FSMerger {
       }
     }
     if (!existingPath.length) {
-      fs.readdir(fullDirPath, options, callback);
+      nodefs.readdir(fullDirPath, options, callback);
     }
     let readComplete = 0;
     for (let i = 0; i < existingPath.length; i++) {
-      fs.readdir(existingPath[i], options, (err: NodeJS.ErrnoException | null , list: string[]) => {
+      nodefs.readdir(existingPath[i], <any>options, (err: NodeJS.ErrnoException | null , list: string[]) => {
```

Comment:
`fs-extra` has different call signatures for `readdir` than node's `fs`. It seemed to me that you were using node's api, so I converted the target instance of these to the node's fs module instead.